package.json

```diff
@@ -63,5 +63,6 @@
     ],
     "importOrderSeparation": true,
     "importOrderSortSpecifiers": true
-  }
+  },
+  "packageManager": "yarn@1.22.19+sha1.4ba7fc5c6e704fce2066ecbfb0b0d8976fe62447"
```

Comment:
Please remove this :D

Reply:
Will remove, was added automatically by corepack. Worth noting though that locking down the package manager (and subsequently having corepack automatically use the right package manager version) is not a bad idea - perhaps we can re-add in another PR?